core/nwb.base.yaml

```
@@ -64,12 +64,10 @@ datasets:
 
 - neurodata_type_def: ExternalImage
   neurodata_type_inc: BaseImage
-  doc: A type for referencing an external image file.
-  attributes:
-  - name: file_path
-    dtype: text
-    doc: Path or URL to the external image file.
-    required: true
+  doc: A type for referencing an external image file. The single file path or URL to the external image file should
+    be stored in the dataset.
```

```suggestion
  doc: A type for referencing an external image file. The single file path or URL to the external image file should
    be stored in the dataset. This type should NOT be used if the image is stored 
     in another NWB file and that file is linked to this file.
```

I noticed this text was included in the `ImageSeries.external_file` description. If I understand correctly, I think the same applies here? It might be helpful to include to clarify whether this type is for external links to other datasets vs. other external file types.

---

That makes sense. Thanks!